app/api_key/rest.py

```diff
@@ -25,3 +29,26 @@ def get_api_key_stats(api_key_id):
     }
     data["total_sends"] = data["email_sends"] + data["sms_sends"]
     return jsonify(data=data)
+
+
+@api_key_blueprint.route('/ranked-by-notifications-created/<n_days_back>', methods=['GET'])
+def get_api_keys_ranked(n_days_back):
+    try:
+        n_days_back = int(n_days_back)
```

Comment:
How much should I protect against bad input here? should I check that `n_days_back` isn't negative?